hermes-console/static/js/console/subscription/SubscriptionController.js

```diff
@@ -82,14 +85,14 @@ subscriptions.controller('SubscriptionController', ['SubscriptionRepository', 'S
           return filtered;
         };
 
-        $scope.edit = function (subscription) {
```

Comment:
As far as I remember, this function is also called from JS code (function `debugFilters` from `SubscriptionController`). I see no changes around that place, so I think it will not work correctly with filters debugger.